webapp/src/components/chat/chat-history/ChatHistoryDocumentContent.tsx

```diff
@@ -81,23 +85,30 @@ export const ChatHistoryDocumentContent: React.FC<ChatHistoryDocumentContentProp
 
     return (
         <>
-            {documents.map((document, index) => (
+            {documents.map(({ name, size, isUploaded }, index) => (
```

Comment:
nit: this would keep things better defined
```suggestion
            {documents.map((document, index) => (
            const { name, size, isUploaded } = document;
```

Reply:
I tend to prefer avoiding multiple accesses of the same property by destructuring the variable, esp when the variable isn't needed. will leave as-is